src/test/groovy/pl/cinema/server/reservation/WeekendPriceEvaluatorSpec.groovy

```diff
@@ -0,0 +1,30 @@
+package pl.cinema.server.reservation
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import spock.lang.Specification
+
+import java.time.LocalDateTime
+
+@SpringBootTest
+//@ContextConfiguration(classes = ServerApplication)
+class WeekendPriceEvaluatorSpec extends Specification {
+
+    private static final NOW = LocalDateTime.now()//clock.fixed() should be used
+
+    @Autowired
+    private WeekendPriceEvaluator objectUnderTest
+
+    def "EvaluatePrice"() {
+    }
+
+    def "canHandle should return true during weekend"() {
+        given:
+            def givenDate = LocalDateTime.of(2019, 7, 6, 12, 0)
+            LocalDateTime.now() >> givenDate
+
+        expect:
+            objectUnderTest.canHandle(null)
```

Comment:
trochę bieda test. Mało scenaraiuszy testowych

Reply:
cóż, nie jest to dobry test, o pewnej godzinie wczorajszego wieczora uznałem, że dodam komentarz //clock.fixed() should be used
i wyśle już wam kod as is. Trzeba by przerobić now wszystkie i uzywać clocka dzięki czemu w tym teście można by jak biały człowiek zamockować odpowiednią godzinę,  używając clock.fixed własnie i w ten sposób to testować.